pkg/kubelet/kuberuntime/labels_test.go

```diff
@@ -214,11 +217,11 @@ func TestContainerAnnotations(t *testing.T) {
 		ContainerPorts:            containerPorts,
 		PodDeletionGracePeriod:    pod.DeletionGracePeriodSeconds,
 		PodTerminationGracePeriod: pod.Spec.TerminationGracePeriodSeconds,
-		Hash:                      kubecontainer.HashContainer(container),
-		HashZeroResources:         kubecontainer.HashContainerZeroResources(container),
-		RestartCount:              restartCount,
-		TerminationMessagePath:    container.TerminationMessagePath,
-		PreStopHandler:            container.Lifecycle.PreStop,
+		Hash:                   kubecontainer.HashContainer(container),
```

Comment:
should the alignment stay as before (aligned to PodTerminationGracePeriod)

Reply:
done by vim go plugin.